### DISCUSSION TITLE
为什么使用github actions配置fluid主题，访问页面一片空白呢？

### DISCUSSION THREAD

**Discussion by Floweryu** [hexojs/hexo#5234] [Question / Answer] [answered]
这是我的仓库目录：
![image](https://github.com/hexojs/hexo/assets/45811382/d0d17f92-bbe7-4caf-9454-03f2bda77eb3)
这是_config.yml文件：
![image](https://github.com/hexojs/hexo/assets/45811382/b13d71be-ba06-4e84-946a-2e851343c3d9)

**Comment by stevenjoezhang** [comment, marked as answer]
看上去问题已解决 https://floweryu.github.io

**Reply by Floweryu** [reply]
嗯，已经解决好了
